tests/resources/Harbor-Util.robot

```diff
@@ -627,3 +630,19 @@ Basic Docker Command With Harbor
     Log  ${output}
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  Untagged
+
+Fetch Harbor Self Signed Cert
+    [Tags]  secret
+    [Arguments]  ${harbor-ip}  ${user}=admin  ${password}=%{TEST_PASSWORD}
+    ${rc}=  Run And Return Rc  wget -q --tries=10 --connect-timeout=10 --auth-no-challenge --no-check-certificate --user ${user} --password ${password} https://${harbor-ip}/api/systeminfo/${harbor_cert}
+    Should Be Equal As Integers  ${rc}  0
+    [Return]  ${harbor_cert}
```

Comment:
like I said above, I think any test which is using this keyword should not be aware of either harbor api or name of this file. so it's passed to others by returning.